src/Console/Schedule.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Console;
+
+use Flarum\Foundation\Config;
+use Illuminate\Console\Scheduling\Schedule as LaravelSchedule;
+use Illuminate\Support\Collection;
+
+class Schedule extends LaravelSchedule
+{
+    public function dueEvents($container)
+    {
+        return (new Collection($this->events))->filter->isDue(new FakeApp($container));
+    }
+}
+
+class FakeApp
```

Comment:
Multiple class declarations in one file are really not done. It makes discoverability extremely hard. Either:

- move that class into its own file
- or use an anonymous class

Reply:
I keep forgetting anonymous classes exist. This is the perfect use case for one. I'll open a PR to clean this up, thanks!